databuilder/databuilder/transformer/complex_type_transformer.py

```diff
@@ -33,12 +33,12 @@ def init(self, conf: ConfigTree) -> None:
             module_name, function_name = parsing_function.rsplit(".", 1)
             mod = importlib.import_module(module_name)
             self._parsing_function = getattr(mod, function_name)
-        except Exception as e:
-            LOGGER.error(f"Invalid parsing function provided to ComplexTypeTransformer: {e}")
+        except (ValueError, ModuleNotFoundError, AttributeError):
+            raise Exception("Invalid parsing function provided to ComplexTypeTransformer")
```

Comment:
even better is to do nothing. this erases information about the actual exception that occurred and instead raises a generic exception with a generic message.